model/adapter.py

```diff
@@ -723,6 +725,32 @@ def query_fun(rxn):
             reaction.bounds = 0, upper_bound
         return reaction
 
+    def minimize_distance(self):
+        """Replaces fluxomics measurements with the minimized distance"""
+        reaction_measurements = [m for m in self.measurements if m['type'] == 'reaction']
+        observations = {m['id']: np.mean(m['measurements']) for m in reaction_measurements}
```

Comment:
I don't know if that happens in our data but `np.mean` will return `nan` if any data points are `nan`. In case that happens we should rather use `np.nanmean`.

Reply:
Good point, but do we have any reason to expect `nan`s? I would maybe lean towards the `-to-model` layer having the responsibility to filter those out.

Reply:
Let's ignore it for now. In (a hopefully not so far) future, we should probably build a service that can select experimental data based on the tags and other factors and it would be that service's responsibility to normalize the data, i.e., directly return mean and variance estimates.

Reply:
Discovered that tests do include measurements with `nan` values, so these are now handled after all.